examples/kepler-integration/src/app.js

```diff
@@ -161,38 +163,40 @@ class App extends Component {
 
   render() {
     return (
-      <ThemeProvider theme={theme}>
-        <GlobalStyle
-          // this is to apply the same modal style as kepler.gl core
-          // because styled-components doesn't always return a node
-          // https://github.com/styled-components/styled-components/issues/617
-          ref={node => {
-            if (node) {
-              this.root = node;
-            }
-          }}
-        >
-          <WindowSize>
-            <ExportVideo />
-            <div style={{transition: 'margin 1s, height 1s', flex: 1}}>
-              <AutoSizer>
-                {({height, width}) => (
-                  <KeplerGl
-                    mapboxApiAccessToken={AUTH_TOKENS.MAPBOX_TOKEN}
-                    id="map"
-                    /*
-                     * Specify path to keplerGl state, because it is not mount at the root
-                     */
-                    getState={keplerGlGetState}
-                    width={width}
-                    height={height}
-                  />
-                )}
-              </AutoSizer>
-            </div>
-          </WindowSize>
-        </GlobalStyle>
-      </ThemeProvider>
+      <IntlProvider locale="en" messages={messages.en}>
+        <ThemeProvider theme={theme}>
+          <GlobalStyle
+            // this is to apply the same modal style as kepler.gl core
+            // because styled-components doesn't always return a node
+            // https://github.com/styled-components/styled-components/issues/617
+            ref={node => {
+              if (node) {
+                this.root = node;
+              }
+            }}
+          >
+            <WindowSize>
+              <ExportVideo />
+              <div style={{transition: 'margin 1s, height 1s', flex: 1}}>
+                <AutoSizer>
+                  {({height, width}) => (
+                    <KeplerGl
```

Comment:
Kepler would internally provide it's own IntlProvider. I guess these don't conflict (or kepler overrides the incoming object?)

Reply:
Yeah the inner one clobbers the outer one, so this should be fine. It's mainly there to make the modal work even though it's not embedded inside Kepler itself.